docs/auth.md

```diff
@@ -0,0 +1,37 @@
+# Auth
+
+## Authentication
+
+### OpenID Provider (OIDC)
+
+`kops` allows you to set the relevant kube-apiserver options through the cluster yaml (`kops edit cluster`).
+
+Here is an example setting up OIDC with Google:
+
+```
+spec:
+  kubeAPIServer:
+    oidcClientID: 766499101111-4coma6ekm47gg21k1xxjd9gbuh541q5r.apps.googleusercontent.com
+    oidcIssuerURL: https://accounts.google.com
+    oidcUsernameClaim: email
+```
+
+You can find more here - [Kubernetes - Authenticating](http://kubernetes.io/docs/admin/authentication/)
+
+## Authorization
+
+### RBAC
+
+`kops` allows you to set the relevant kube-apiserver options through the cluster yaml (`kops edit cluster`).
+
+Here is an example setting up RBAC:
+
+```
+spec:
+  kubeAPIServer:
+    runtimeConfig:
+      - rbac.authorization.k8s.io/v1alpha1: true
+    authorizationMode: RBAC
```

Comment:
Sorry i have left this hanging. I have not been able to this yet, hoping to get to it this week. My assumption is that the policies can be defined after provisioning the cluster as the rules will differ between users. However I have seen on Justin's branch it includes policies for cluster components (e.g. kubelet), so not sure if these have to be on place for the cluster to work at all